src/Admin/src/RoutesDelegator.php

```diff
@@ -16,12 +28,33 @@ public function __invoke(ContainerInterface $container, string $serviceName, cal
         /** @var Application $app */
         $app = $callback();
 
-        $app->route(
-            '/admin[/{action}[/{uuid}]]',
-            AdminController::class,
-            [RequestMethodInterface::METHOD_GET, RequestMethodInterface::METHOD_POST],
-            'admin'
-        );
+        $app->get('/admin/add', GetAdminFormHandler::class, 'admin::get-admin-form');
+        $app->post('/admin/add', CreateAdminHandler::class, 'admin::create-admin');
+
+        $app->get('/admin/edit/{uuid}', GetEditAdminFormHandler::class, 'admin::get-edit-admin-form');
+        $app->post('/admin/edit/{uuid}', EditAdminHandler::class, 'admin::edit-admin');
+
+        $app->get('/admin/delete/{uuid}', GetDeleteAdminFormHandler::class, 'admin::get-delete-admin-form');
+        $app->post('/admin/delete/{uuid}', DeleteAdminHandler::class, 'admin::delete-admin');
+
+        $app->get('/admin/list', AdminListHandler::class, 'admin::list');
+        $app->get('/admin/logins', AdminListLoginHandler::class, 'admin::list-logins');
+
+        $app->get('/admin/account', GetAccountFormHandler::class, 'admin::account');
+        $app->post('/admin/account', EditAccountHandler::class, 'admin::edit-account');
+
+        $app->post('/admin/change-password', ChangePasswordHandler::class, 'admin::change-password');
```

Comment:
```suggestion
        $app->post('/admin/my-account/edit-password', EditMyAccountPasswordHandler::class, 'admin::edit-my-account-password');
```
This might look a bit weird, but it tries to be in tone with the pattern followed by the other routes.

Reply:
I choose `/account/`  instead of  `/my-account/`. We can discuss this further if necessary.